### DISCUSSION TITLE
Ideas, Questions, Anything

### DISCUSSION THREAD

**Discussion by mechatroner** [mechatroner/sublime_rainbow_csv#30] [General]
Hello World!

**Comment by cederholm** [comment]
Great plugin 😊. Some column are shown in italic. Why? Can this be avoided?

![Screenshot 2021-11-06 at 18 04 27](https://user-images.githubusercontent.com/39326957/140617871-a61a2b73-a22f-44b9-91a7-e8a598a52f9e.png)

**Comment by mechatroner** [comment]
@cederholm This seems to be happening with 7-th and 10-th columns and so this is because these columns are based on `storage.type` and `variable.language` scopes that are mapped to styles with italic property. To avoid this you can put a custom color scheme file here: `Packages/User/RainbowCSV.sublime-color-scheme` , see this instruction for details: https://github.com/mechatroner/sublime_rainbow_csv/blob/master/README.md#auto_adjust_rainbow_colors

**Comment by cederholm** [comment]
Dmitry Ignatovich,

Perfect - thank you!
